### DISCUSSION TITLE
It is possible to select and copy text from a read-only editor

### DISCUSSION THREAD

**Discussion by FelixMittermeier** [singerdmx/flutter-quill#695] [Web]
My issue is about Web, but most likely are all platforms affected.

With the latest version it is possible that one can select and copy the editor content for example with STR+A and SRG+C even if the editor is in read-only mode and the selection tools have been disabled by passing the according boolean values to the quill editor. This didn't happen before the changes needed for flutter 2.10 so it somehow looks like a regression.

**Comment by FelixMittermeier** [comment]
Why was my bug report converted into a discussion?
It's not a question. The named action is possible while it shouldn't be possible to do this.

**Comment by singerdmx** [comment]
Oh sorry for converting.
Is this issue still existing?

**Comment by singerdmx** [comment]
I kind of don't fully request the ask here.
I think in readonly mode, one should be able to copy things.
